src/components/file-uploader/_file-uploader.scss

```diff
@@ -152,13 +161,42 @@
     align-items: center;
     justify-content: space-between;
     min-height: rem(30px);
-    background-color: $inverse-01;
+    background-color: $ibm-colors__gray-10;
     padding: 0 $spacing-xs 0 $spacing-md;
-    margin-bottom: $spacing-sm;
+    margin-bottom: $spacing-xs;
+    overflow-y: hidden;
 
     &:last-child {
       margin-bottom: 0;
     }
+
+    .#{$prefix}--inline-loading__animation,
+    .#{$prefix}--loading {
+      width: 1.5rem;
+      height: 1.5rem;
+    }
+
+    .#{$prefix}--inline-loading__animation {
+      right: -0.25rem; // offset due to larger SVG size
+    }
+  }
+
+  .#{$prefix}--file__selected-file--invalid {
+    @include focus-outline('invalid');
+    margin-bottom: $spacing-2xs;
+
+    .#{$prefix}--file-filename {
+      color: $ibm-colors__gray-30;
```

Comment:
@laurenmrice Is there a theme token that should be applied to invalid filename's color? Thanks!

Reply:
Hey Akira! The type style token is caption-01 and the color token for the invalid text is support-01.